src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,41 @@
+.catalog {
+  display: grid;
+
```

Comment:
it might be better to include "grid-template-columns: repeat(1, auto)" here for clarity, so it would be easier to understand code logic from 1 column to 4, even though it works just fine without it